lib/clamp-link.in

```diff
@@ -108,8 +108,7 @@ done
 
 
 
-
-for ARG in "$@"
+for ARG in "${INPUT_ARGUMENTS[@]}"
```

Comment:
INPUT_ARGUMENTS is not defined in the TIP probably because you earlier linker patch was reverted, could you take a look at this?